website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Project migration guide for single API version operator
+linkTitle: Project Migration - Single API version operator
+weight: 1
+---
+
+This guide walks through the steps required to migrate an operator built using Operator SDK to kubebuilder project layout.
+
+The document considers [memcached operator][memcached-operator] as an example to describe the process of migrating an operator built using Operator SDK to a runnable Kubebuilder project.
+
+## Prerequisites
+- [git][git_tool].
+- [go][go_tool] version 17.03+.
+- [kubectl][kubectl_tool] version v1.11.3+.
+- [kustomize][kustomize_tool] v3.1.0+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+## Install Kubebuilder
+
+Follow the installation steps provided in the [kubebuilder book][kubebuilder_install] to install kubebuilder.
+
+## Create a new project
+
+Use [`kubebuilder init`][kubebuilder-create-project] command to create a new project:
+
+```sh
+$ mkdir $HOME/projects
+$ cd $HOME/projects
+$ kubebuilder init --domain example.com --license apache2 --owner "The Memcached Operator authors" --repo github.com/example-inc/memcached-operator
+$ cd memcached-operator
+```
+
+Make sure to activate the go module support by running `$ export GO111MODULE=on`.
+
+## Create an API
+
+Create a new API (group/version) as `cache/v1alpha1` and a new kind(CRD) `Memcached` on it.
```

Comment:
I think we could rephrase the migration steps a little bit.
Currently the steps in this section and the next directly state what API to define and then to copy over our example spec and status. This reads more like the user-guide.

For the migration guide I think we could probably rephrase each step into two parts
```
- Do XYZ for your project.
- Here's how XYZ looks like for the memcached operator example.
```